tests/services/pids/providers/test_datacite_pid_provider.py

```diff
@@ -195,24 +195,55 @@ def custom_format_func(*args):
     assert datacite_provider.create(record).pid_value == expected_result
 
 
-def test_datacite_provider_validation(record, mocker):
-    client = DataCiteClient("datacite")
-
-    # check with default func
+def test_datacite_provider_validate(record):
     current_app.config["DATACITE_PREFIX"] = "10.1000"
+    client = DataCiteClient("datacite")
     datacite_provider = DataCitePIDProvider("datacite", client=client)
+
+    # Case - Valid identifier (doi)
     success, errors = datacite_provider.validate(
         record=record, identifier="10.1000/valid.1234", provider="datacite"
     )
     assert success
-    assert errors == []
+    assert [] == errors
```

Comment:
(pet peeve: pytest shows the results of comparison tests like this with left-hand side as the expected result, so we should do the same to make reading test errors easier)